websocket/polygon.go

```diff
@@ -136,6 +165,9 @@ func (c *Client) setTopic(topic Topic) {
 }
 
 func (c *Client) Subscribe(topic Topic, tickers ...string) error {
+	c.mtx.Lock()
```

Comment:
need to guard these methods since we push to the write queue (which could be cleared during reconnects)